plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -400,6 +394,14 @@ jQuery( function ( $ ) {
 		.find( '.woocommerce_attribute' )
 		.get();
 
+	// If the product has no attributes, add an empty attribute to be filled out by the user.
+	$( function add_blank_custom_attribute_if_no_attributes() {
```

Comment:
This function doesn't necessarily need to be named, especially if it's only being used in this one place.

Reply:
Naming functions is very helpful when you are looking at an outline of the code in an editor, debugging a stack trace, and to help document what the code is doing.